ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-stable-4.12.yaml

```diff
@@ -431,6 +431,24 @@ tests:
     - ref: cucushift-installer-check
     - chain: openshift-upgrade-qe-test
     workflow: cucushift-installer-rehearse-azure-upi
+- as: baremetal-ha-agent-ipv4-static-disconnected-p1-f7
```

Comment:
Do we need `UPGRADE_PRE/CHECK_RUN_TAGS` for this ? I can see aws test cases are executed in the log.